### ISSUE TITLE
Sometimes search incorrectly returns no results

### ISSUE THREAD

**Issue by LH-Lawliet** [DrKain/scrape-youtube#21]
**Describe the bug**
Hy, I don't know if it's really a bug or if there is any reason to that but sometime there is no video in the result, but for the same search in youtube, there is some result

**To Reproduce**
```javascript
youtube.search("l'enfant sauvage").then(results => {
    console.log(results)
});
```

So the expected result is something with like 20 videos
But i got `{videos: Array(0), playlists: Array(0), streams: Array(0)}`

With most of my research there is no error, but there is some with request containing a certain word like:
l'enfant sauvage
Hero Of War
Bombtrack
Incense
Thunderstruck

**Versions:**
 - Package: 7.0.8 (npm -v youtube-scraper)
 - Node: 15.2.1

**Comment by DrKain** [comment]
This appears to be an issue with google ads and is unrelated to the search query, I'll look into this problem.

**Comment by DrKain** [comment]
This should now be fixed in [scrape-youtube@2.0.5](https://www.npmjs.com/package/scrape-youtube)
